modules/nf-core/regtools/junctionsextract/tests/main.nf.test

```diff
@@ -0,0 +1,63 @@
+// nf-core modules test regtools/junctionsextract
+nextflow_process {
+
+    name "Test Process REGTOOLS_JUNCTIONSEXTRACT"
+    script "../main.nf"
+    process "REGTOOLS_JUNCTIONSEXTRACT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "regtools"
+    tag "regtools/junctionsextract"
+
+    test("homo_sapiens - bam-bai") {
+
+        when {
+            process {
+                """
+
+                input[0] = [
+                    [ id:'test', single_end:false ], // meta map
+                    file(params.test_data['homo_sapiens']['illumina']['test_rna_paired_end_sorted_chr6_bam'], checkIfExists: true),
+                    file(params.test_data['homo_sapiens']['illumina']['test_rna_paired_end_sorted_chr6_bam_bai'], checkIfExists: true)
```

Comment:
Can you swap these to the more recent full file path please.

Reply:
fixed in commit [more recent fill file path](https://github.com/nf-core/modules/pull/7033/commits/a6d4cc520a666b0ce0dcf7706311b8b5922550bf) (sorry, typo, fill should be full)